etc/schema.json

```diff
@@ -3312,6 +3312,135 @@
                 },
                 "version": {
                     "type": "string"
+                },
+                "sdp": {
+                    "type": "object",
+                    "description": "SDP message body",
+                    "optional": true,
+                    "properties": {
+                        "version": {
+                            "type": "integer",
+                            "description": "SDP protocol version"
+                        },
+                        "origin": {
+                            "type": "string",
+                            "description": "Owner of the session"
+                        },
+                        "session_name": {
+                            "type": "string",
+                            "description": "Session name"
+                        },
+                        "session_info": {
+                            "type": "string",
+                            "optional": true,
+                            "description": "Textual information about the session"
+                        },
+                        "uri": {
+                            "type": "string",
+                            "optional": true,
+                            "description": "A pointer to additional information about the session"
```

Comment:
Very nit, and can be ignored if this is ready to merge. I imagine that this choice of words is to conform with the RFC description, but wonder if using a description like the one in the code comments ("URI of description") or something like "A URI with additional information about the section" or "A URI with session description" isn't more informative for users?